wallet-api/wallet-api.cabal

```diff
@@ -38,6 +38,7 @@ library
         Ledger.Types
         Ledger.Validation
         Ledger.Index
+        Language.PlutusTx.Interval
```

Comment:
TBH I'd just call it `Ledger.Interval`. It certainly has a bunch of TH, but it's not inherently PlutusTx specific.

Reply:
done